bot/extensions/fun/weather.py

```diff
@@ -0,0 +1,129 @@
+from geopy.geocoders import Nominatim
+from timezonefinder import TimezoneFinder
+import pytz
+from pytz import country_timezones, timezone
+from datetime import datetime
+from discord.ext.commands import Cog, command
+import requests
+from discord import Embed
+import time
+import string
+
+from discord.utils import parse_time
+
+class WeatherCog(Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='weather', help='Show weather information in your city', usage="{city}")
+    async def my_command(self, ctx, *city_input):
+        # get all argument inputed by the user 
+        # on discord and join them together 
+        # (handle double-words/argument cities)
+        city = " "
+        city = string.capwords(city.join(city_input))
+    
+        # initialize Nominatim API
+        geolocator = Nominatim(user_agent="geoapiExercises")
+        # city_name = city.capitalize()
+        # getting Latitude and Longitude
+        location = geolocator.geocode(city)
+    
+        # pass the Latitude and Longitude
+        # into a timezone_at
+        # and it return timezone
+        obj = TimezoneFinder()
```

Comment:
Find a better name than obj to represent this. Maybe something like `timezone_finder` ber good.